lib/new_relic/agent.rb

```diff
@@ -293,6 +294,46 @@ def notice_error(exception, options = {})
       nil # don't return a noticed error data structure. it can only hurt.
     end
 
+    # Set a callback proc for determining an error's error group name
+    #
+    # @param [Proc] the callback proc
+    #
+    # Typically this method should be called only once to set a callback for
```

Comment:
Should you mention that it should be called when the app initializes, but after the agent has loaded? Or does call time not matter?

Reply:
Call time does not matter. Essentially, as long as Ruby doesn't error out saying that it doesn't know what `NewRelic::Agent` is, you're ready to call it. We're just populating an instance var with the result of the call, so we're ready immediately.